marlowe-playground-client/src/Marlowe/Gen.purs

```diff
@@ -35,11 +35,11 @@ genRational = do
   d <- genBigInteger
   pure
     -- we need to do this because in tests where we wrap a Rational in a Term or TermWrapper
-
+    
     -- when we have two negative values then the column position of the term will different
-
+    
     -- to if we have two positive values, even though the rationals themselves are equal
-
+    
```

Comment:
where did these come from? has `$(nix-build ../default.nix -A dev.scripts.fixPurty)/bin/fix-purty` been run?

Reply:
I believe that's exactly the result of fix-purty

Reply:
cool, as long as it comes from a tool and not a human